app/Repositories/SignupRepository.php

```diff
@@ -37,28 +29,17 @@ public function create($data)
         // Let Laravel take care of the timestamps unless they are specified in the request
         // @TODO: keep only the else after the migration
         if (isset($data['created_at'])) {
-            // Associate the event with the signup
-            $event->signup_id = $signup->id;
-
             // Set the created_at time
             $signup->created_at = $data['created_at'];
-            $event->created_at = $data['created_at'];
 
             // Set the updated time if provided, if not, assume no updates
             if (isset($data['updated_at'])) {
                 $signup->updated_at = $data['updated_at'];
-                $event->updated_at = $data['updated_at'];
             } else {
                 $signup->updated_at = $data['created_at'];
-                $event->updated_at = $data['created_at'];
             }
 
             $signup->save(['timestamps' => false]);
```

Comment:
The signup gets backdated after it gets created, so I don't think that would have the right timestamps. Unless there is a way to create a model with `'timestamps' => false` which I didn't think there was without turning off timestamps on the model altogether